### DISCUSSION TITLE
podman "Cannot open TUN/TAP dev /dev/net/tun: No such device (errno=19)"

### DISCUSSION THREAD

**Discussion by pierrepaap** [haugene/docker-transmission-openvpn#2661] [Q&A]
Hey everyone,
Looks like an old bug has crept back up.
Before calling it a bug i'll submit my issue here.

I'm running the container with podman on an Arch Linux system.
it worked fine until last night. When the system booted today, the container failed to restart (probably because of an automatic update)

Here is an extract of the logs:
`Starting container with revision: 1103172c3288b7de681e2fb7f1378314f17f66cf`
`TRANSMISSION_HOME is currently set to: /config/transmission-home`
`Creating TUN device /dev/net/tun`
`Using OpenVPN provider: <redacted>`
[...]
`2023-07-16 17:34:45 ERROR: Cannot open TUN/TAP dev /dev/net/tun: No such device (errno=19)`
`2023-07-16 17:34:45 Exiting due to fatal error`

Unsurpisingly, I already have the proper arguments when calling the container:
`/usr/bin/podman run \`
[...]
`	--privileged \`
[...]	
`	-e CREATE_TUN_DEVICE=true \`
[...]
`	-v /etc/localtime:/etc/localtime:ro docker.io/haugene/transmission-openvpn`

I can check other stuff, but at this point this is what I knew to do.
Any ideas?
if it's a bug i'll register full info of course (logs, config...)

**Comment by pkishino** [comment]
Hmm, can’t find that revision.. is it latest? Try :dev or :5.0.2 tag

**Reply by pierrepaap** [reply]
yes it's supposed to be latest. I'll try 5.0.2

**Reply by pierrepaap** [reply]
tried with :dev and it worked
`Starting container with revision: 42eb2ee94ef9a3ce45bdccb308f9387b36c4f6e0`

then went back to latest (initially explicitly with :latest then implicitly) and it also worked
`Starting container with revision: 1103172c3288b7de681e2fb7f1378314f17f66cf`

So, I'm baffled and evrything is back to normal
The default UI changed though 

Thanks anyways !

**Comment by pkishino** [comment]
Most have been using a very old image

**Reply by pierrepaap** [reply]
Sooooo
same player plays again.
Same error again today, I tried latest, I tried dev and I tried removing the images and starting again (bothe latest and dev).
To no avail, the error has returned, to my bafflement again.

**Reply by pierrepaap** [reply]
After stopping the systemd service, rebooting and starting the systemd service manually it works again. 
Something is off, i just don't know what
